ui/src/timeMachine/selectors/index.test.ts

```diff
@@ -62,88 +58,3 @@ describe('TimeMachine.Selectors.Index', () => {
     expect(getEndTime(pastThirtyDaysTimeRange)).toBeGreaterThanOrEqual(now)
   })
 })
-
-describe('getting active tag values', () => {
-  const activeQueryTags = [
-    {
-      keys: [
-        '_field',
-        '_measurement',
-        'cpu',
-        'device',
-        'fstype',
-        'host',
-        'interface',
-        'mode',
-        'name',
-        'path',
-      ],
-      values: [
-        'cpu',
-        'disk',
-        'diskio',
-        'mem',
-        'net',
-        'processes',
-        'swap',
-        'system',
-      ],
-    },
-    {
-      keys: ['_field', 'host'],
-      values: [
-        'active',
-        'available',
-        'available_percent',
-        'buffered',
-        'cached',
-        'commit_limit',
-        'committed_as',
-        'dirty',
-        'free',
-        'high_free',
-        'high_total',
-        'huge_page_size',
-        'huge_pages_free',
-        'huge_pages_total',
-        'inactive',
-        'low_free',
-        'low_total',
-        'mapped',
-        'page_tables',
-        'shared',
-        'slab',
-        'swap_cached',
-        'swap_free',
-        'swap_total',
-        'total',
-        'used',
-        'used_percent',
-        'vmalloc_chunk',
-        'vmalloc_total',
-        'vmalloc_used',
-        'wired',
-        'write_back',
-        'write_back_tmp',
-      ],
-    },
-    {
-      keys: ['host'],
-      values: ['foo_computer'],
-    },
-  ]
-
-  it('returns the active query tag values when the function is filter', () => {
-    const actualTags = getActiveTagValues(activeQueryTags, 'filter', 2)
-    expect(actualTags).toEqual(activeQueryTags[2].values)
-  })
-
-  it('returns all previous tag values when the function is group', () => {
-    const actualTags = getActiveTagValues(activeQueryTags, 'group', 2)
-    expect(actualTags).toEqual([
-      ...activeQueryTags[0].values,
-      ...activeQueryTags[1].values,
-      ...activeQueryTags[2].values,
-    ])
-  })
-})
```

Comment:
this test was testing the wrong thing

Reply:
can we test the right thing?

Reply:
>can we test the right thing?
@121watts only cause you asked https://github.com/influxdata/influxdb/pull/16992/files#diff-c432c510f4c140978dbe371ff3e9142dR93